src/cloudfront.rs

```diff
@@ -35,17 +35,21 @@ impl CloudFront {
     /// Invalidate a file on CloudFront
     ///
     /// `path` is the path to the file to invalidate, such as `config.json`, or `re/ge/regex`
-    #[instrument(skip(self))]
     pub async fn invalidate(&self, path: &str) -> anyhow::Result<()> {
-        let path = if path.starts_with('/') {
-            path.to_string()
-        } else {
-            format!("/{path}")
-        };
```

Comment:
do we not need this logic anymore? are we sure that all invalidated paths always start with `/`?

Reply:
Good catch! Not sure enough; I'll reinstate it.